modules/tinymce/src/core/test/ts/browser/dom/ScrollIntoViewTest.ts

```diff
@@ -24,6 +24,7 @@ describe('browser.tinymce.core.dom.ScrollIntoViewTest', () => {
   const hook = TinyHooks.bddSetup<Editor>({
     add_unload_trigger: false,
     height: 500,
+    with: 1000,
```

Comment:
I assume this should be `width`. I guess it passes since the viewport width is less than 2000px but setting an explicit width would make sure that the test doesn't fail on very wide monitors.

Reply:
yes, it was to be sure that the value in the test was always greater than the width of the editor, I fixed the typo